src/udiskslinuxpartition.c

```diff
@@ -179,6 +179,39 @@ check_authorization (UDisksPartition       *partition,
 
 /* ---------------------------------------------------------------------------------------------------- */
 
+static void update_partitions_list (UDisksObject           *disk_object,
+                                    UDisksLinuxBlockObject *part_object)
+{
+  UDisksPartitionTable *table = NULL;
+  gchar **partitions = NULL;
+  const gchar *object_path = NULL;
+  guint num_parts = 0;
+
+  object_path = g_dbus_object_get_object_path (G_DBUS_OBJECT (part_object));
+  if (object_path == NULL)
+    goto out;
+
+  table = udisks_object_peek_partition_table (disk_object);
+  if (table == NULL)
+    goto out;
```

Comment:
These two `goto` can really be just `return`s.

Reply:
Fixed.